storage/kv_test.go

```diff
@@ -194,10 +180,8 @@ func testKVRangeBadRev(t *testing.T, f rangeFunc) {
 	s := NewStore(b, &lease.FakeLessor{})
 	defer cleanup(s, b, tmpPath)
 
-	s.Put([]byte("foo"), []byte("bar"), lease.NoLease)
-	s.Put([]byte("foo1"), []byte("bar1"), lease.NoLease)
-	s.Put([]byte("foo2"), []byte("bar2"), lease.NoLease)
-	if err := s.Compact(3); err != nil {
+	putTestKVs(s)
```

Comment:
can we give this a more descriptive name? like Put3TestKVs or whatever? PutTestKVs seems to be so generic.